BenadrylAutomation/src/main/java/com/jnj/pageobjects/SignUpPage.java

```diff
@@ -71,7 +70,20 @@ public void verifyLogoAndTitle(String expectedTitle) {
 		} catch (Exception e) {
 			Assert.fail("Logo not displayed");
 		}
+	}
 
+	/**
+	 * Function to click on close button of signup pop-up
+	 *
+	 */
+	public void closeSignupPopup() {
+		Action.waitFor(5000);
```

Comment:
@vaibhavNagvekarSJI replace the above method with the one which I have sent you

Reply:
fixed

Reply:
Verified @vaibhavNagvekarSJI